### ISSUE TITLE
Allow destroy on messages

### ISSUE THREAD

**Issue by Ehesp** [ant-design/ant-design#902]
Hi

I noticed that on the `message` component you cannot remove one which is already visible. However on the `rc-notification` package there is a `destory()` method, but it doesn't seem to be exposed on this lib?

Any chance of allowing it?

**Comment by afc163** [comment]
You can get `close` function from the return value.

```js
const closeMessage = message.success('message', 100);
closeMessage();
```

**Comment by afc163** [comment]
Ok, I see... You ask for `destory` not the `close`. I can add it.

**Comment by afc163** [comment]
Fixed in branch `develop-0.12.0`.

**Comment by Ehesp** [comment]
:+1:
